### PR TITLE
Move "Add layer" button from dropdown to header

### PR DESCRIPTION
**Pull request by rmartz**
The button is now labeled "Browse" and appears between the search bar
and "Annotate". This also moves some functionality from the options_menu
view to header to drive displaying the add layer screen.

### CODE REVIEW

**Comment by CloudNiner** [comment]
Screenshot?

File: app/views/admin/shared/_vis_header.html.erb [hunk 1]
```diff
@@ -9,11 +9,12 @@
               <input type="text" />
               <input type="submit" />
           </li></form>
+          <li><a class="rounded button add_new_layer" href="#/add-new-layer">Browse</a></li>
```
Comment (CloudNiner) [inline app/views/admin/shared/_vis_header.html.erb:12]: Shouldn't the button text be "Add Layer"?

**Comment by rmartz** [comment]
Screenshot:
screen shot 2016-07-13 at 1 26 32 pm

The "Add a new layer" window that appears when clicked isn't changed so I'm not including that in this PR.

File: app/views/admin/shared/_vis_header.html.erb (hunk 1 above)
Comment (rmartz) [inline app/views/admin/shared/_vis_header.html.erb:12]: My take was that the button was in the wireframes but got missed, and the wireframes I looked at on #44 and #45 showed the button between the search bar and "Annotate" marked as "Browse". Is there a more appropriate place to look?

**Comment by CloudNiner** [comment]
+1 consider updating button text, although I don't recall what @mbforr wanted it to be from the meeting this am.

**Comment by CloudNiner** [comment]
Fixup the tests too before merging.

File: app/views/admin/shared/_vis_header.html.erb (hunk 1 above)
Comment (CloudNiner) [inline app/views/admin/shared/_vis_header.html.erb:12]: Nope nvmnd, should be "Browse" as discussed.

**Comment by jbranigan** [comment]
"Browse" is what was requested. Looking pretty good.

**Pull request merged**